### PR TITLE
Move --repository in version command to the end

### PR DESCRIPTION
**Pull request by mdellweg**
Do i need an issue to add a changelog entry? Is the mechanism there already in place?

### CODE REVIEW

**Comment by daviddavis** [comment]
Here's your issue: https://github.com/pulp/pulp-cli/issues/123 :)

File: CHANGES/123.removal [hunk 1]
```diff
@@ -0,0 +1 @@
+Moved the location of the `--version` command to the scope of the last command.
```
Comment (daviddavis) [inline CHANGES/123.removal:1]: Doesn't this change move `--repository` to the end?

Comment (mdellweg) [reply CHANGES/123.removal:1]: After the last command at least. There is no specific order of options.
How can i word it better?

Comment (daviddavis) [reply CHANGES/123.removal:1]: What about "Moved the location of \`--repository\` option after the last command"?

**Review by daviddavis — APPROVED** [review]

**Pull request merged**